docs/Contributing/API-for-contributors.md

```diff
@@ -2536,6 +2597,13 @@ Lists the software installed on the current device.
     {
       "id": 121,
       "name": "Google Chrome.app",
+      "available_for_install" true,
+      "software_package": {
+        "name": "GoogleChrome.pkg"
+        "version": "125.12.2"
+      },
+      "app_store_app": null,
+      "self_service": true,
```

Comment:
@mna For `GET /device/:device_token/software`, we want to make it return same response as `GET /hosts/:id/software` to make it easier to maintain and understand.

Currently this endpoint when `self_service` param is included displays `package`. We want to change that and display `software_package` and `app_store_app` all the time. When `self_service` filter is active only each software that has `available_for_install` and `self_service` as `true` will be returned.

@ghernandez345 This requires some changes on the frontend to consider new response.